app/views/DR/News/Details.js

```diff
@@ -70,32 +65,17 @@ const Details = ({
           style={{
             width: Dimensions.get('window').width,
             height: Dimensions.get('window').height,
-            backgroundColor: 'white',
+            backgroundColor: Colors.WHITE,
           }}
-          activityIndicator={activityIndicatorLoadingView(isSponsorsScreen)}
+          activityIndicator={activityIndicatorLoadingView(true)}
         />
       )}
       {switchScreenTo === 'WebView' && (
         <WebView
           source={source}
           startInLoadingState
           injectedJavaScript={fixerImage}
-          renderLoading={() => activityIndicatorLoadingView(isSponsorsScreen)}
-          onLoadStart={() => {
-            if (textFirstDialog) {
-              setShowDialog(true);
-              setDialogText(textFirstDialog);
-            }
-            setNewTimer(
-              setTimeout(() => {
-                setDialogText(t('label.dialog_interval_advice'));
-                setShowDialog(true);
-              }, 30000),
-            );
-          }}
-          onLoadEnd={() => {
-            clearTimeout(timer);
-          }}
+          renderLoading={() => activityIndicatorLoadingView()}
           onError={() => {
             setDialogText(t('label.dialog_error_advice'));
             setShowDialog(true);
```

Comment:
I think you can delete this

Reply:
is good to have an error handler, since the news open in a webView